src/assets/locales/ja.json

```
@@ -513,7 +513,7 @@
   "revoke-token-modal.title": "トークンの取り消し",
   "room-entry-modal.enter-on-device-button": "デバイスで入室",
   "room-entry-modal.join-room-button": "入室",
-  "room-entry-modal.options-button": "オプション",
+  "room-entry-modal.room-settings-button": "部屋の設定",
```

```suggestion
  "room-entry-modal.room-settings-button": "ルームの設定",
```

Translation itself from "room" into "部屋" is correct. But in this file "room" is translated into "ルーム" at a lot of places. So, as consistency I would like to suggest "ルーム" here.